src/scripts.js

```diff
@@ -5,7 +5,7 @@ import './styles.css'
 import { makeTagActive, closeRecipe, showRecipe, searchForRecipes, switchView } from './domUpdates';
 import { calculateRecipeCost, getIngredientAmounts, getInstructions } from './recipes'; 
 import { ingredientsData } from './data/ingredients';
-import { assignCurrentUser, fetchRecipes } from './apiCalls';
+import { dataLoad, pageData } from './apiCalls';
```

Comment:
looks like `pageData` is no longer needed in scripts?

Reply:
I am using it in scripts in my branch! Do we mind leaving it there?

Reply:
removed